examples/Clutter/clutter-example.hs

```diff
@@ -0,0 +1,65 @@
+{-# LANGUAGE OverloadedLabels, OverloadedStrings, ImplicitParams #-}
+module Main where
+
+import GI.GObject (get, AttrOp((:=), On), new)
```

Comment:
It would be better to use `import Data.GI.Base (get, AttrOp((:=), On), new)` here. This way you can also omit the `gi-gobject` build depends in `clutter-example.cabal`.